Source/Core/DolphinQt/Config/Graphics/HacksWidget.cpp

```diff
@@ -151,6 +151,9 @@ void HacksWidget::ConnectWidgets()
 
 void HacksWidget::LoadSettings()
 {
+  // Need to enable the slider if it was turn off (in case of changes)
```

Comment:
```suggestion
  // Re-enable the slider in case it was disabled because of a custom value
```

It might also make sense to move this closer to the `setEnabled(false)` call (perhaps right before the switch statement?)

Reply:
I agree, it could me more clear.